pina/model/layers/embedding.py

```diff
@@ -0,0 +1,142 @@
+""" Embedding modulus. """
```

Comment:
Periodic Boundary Embedding module

Reply:
In the future I plan to put other embeddings, that's why I used this name. What do you think?

Reply:
But I would keep one class per file, it's cleaner